src/main/io/dashboard.c

```diff
@@ -71,11 +72,20 @@
 #include "sensors/sensors.h"
 
 
+PG_REGISTER_WITH_RESET_TEMPLATE(dashboardConfig_t, dashboardConfig, PG_DASHBOARD_CONFIG, 0);
+
+PG_RESET_TEMPLATE(dashboardConfig_t, dashboardConfig,
+    .device = DASHBOARD_I2C_INSTANCE,
```

Comment:
To match bus number `x` to literal representation used in resource command (`I2Cx_{SCL,SDA}`), we have to use ```I2C_DEV_TO_CFG(DASHBOARD_I2C_INSTANCE)```.